provision/vvv-init.sh

```diff
@@ -153,10 +153,9 @@ download_wordpress() {
 
 initial_wpconfig() {
   echo " * Setting up wp-config.php"
-  noroot wp core config --dbname="${DB_NAME}" --dbprefix="${DB_PREFIX}" --dbuser=wp --dbpass=wp  --extra-php <<PHP
-define( 'WP_DEBUG', true );
-define( 'SCRIPT_DEBUG', true );
-PHP
+  noroot wp core config --dbname="${DB_NAME}" --dbprefix="${DB_PREFIX}" --dbuser=wp --dbpass=wp
```

Comment:
Semi-related: `wp core config` is no longer an actual command - it is an alias of `wp config create`.

https://github.com/wp-cli/wp-cli/blob/f94e77d4563c66c04b582eb9fa8b4652b11ed0c2/php/WP_CLI/Runner.php#L733-L736.

Is it worth also updating as well?

Reply:
yes, looks like a good change to make, the same needs to happen in the develop version of the template